lark/parsers/earley.py

```diff
@@ -296,13 +297,13 @@ def parse(self, lexer, start):
                 debug_walker.visit(solutions[0], "sppf.png")
```

Comment:
Optional: create debug images for all solutions

Reply:
Done. 

Might be a bit slow if there are many solutions, but we deal with that when it happens.